tss-esapi/src/context.rs

```diff
@@ -119,6 +110,17 @@ impl Context {
         }
     }
 
+    /// Create a new ESYS context based on the TAB Ressource Manager Daemon.
+    /// The TABRMD will make sure that multiple users can use the TPM safely.
+    ///
+    /// # Errors
+    /// * if either `Tss2_TctiLdr_Initiialize` or `Esys_Initialize` fail, a corresponding
+    /// Tss2ResponseCode will be returned
+    pub fn new_tabrmd(tabrmd_conf: TabrmdConfig) -> Result<Self> {
```

Comment:
@gaetanww specifically this part, this method forces TABRMD usage but is now safe

Reply:
Good to have that method, thanks!
Should it be named something like: “with_tabrmd” or “new_with_tabrmd” or “initalize_with_tabrmd” to conform with rust naming guidelines? 
see https://rust-lang.github.io/api-guidelines/predictability.html#c-ctor

Reply:
I chose `new_with_tabrmd` 🚀